core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -805,6 +806,30 @@ default HttpResponse mapTrailers(Function<? super HttpHeaders, ? extends HttpHea
         return of(stream);
     }
 
+    /**
+     * Applies specified {@link Function} to the non-informational {@link ResponseHeaders}
+     * emitted by this {@link HttpResponse}.
+     *
+     * <p>For example:<pre>{@code
+     * HttpResponse response = HttpResponse.of(ResponseHeaders.of(HttpStatus.OK));
+     * HttpResponse result = res
+     *      .peekHeaders(headers -> assertThat(headers.status()).isEqualTo(HttpStatus.OK));
```

Comment:
We don't use AssertJ in Javadoc because Armeria does not have a transitve dependency for it.
Could you use JDK's API instead?

Reply:
> don't use AssertJ in Javadoc, Could you use JDK's API instead?

oh~ I got it! thanks!